dask_ml/wrappers.py

```diff
@@ -271,18 +293,27 @@ def predict(self, X):
         """
         self._check_method("predict")
         X = self._check_array(X)
+        meta = self.meta
+
+        if meta is None:
```

Comment:
Can you put this warning in the `__init__`? I think that's where scikit-learn typically puts them.

Reply:
Thanks again for the review. I will put that in `__init__`

Reply:
Sorry , I put this in `predict` because this behavior change is only happening in `predict` .  Transform's behavior still remains the same so I think having it here makes much more sense as we are trying to warn users about the upcoming change which only impacts the `predict ` calls. 

Let me know what you think.

Reply:
Understood. The slight concern I have is that the fix for this is to actually set it in init. But agreed that having it here is better, since this won't affect you if you're just using fit.